scripts/update-ts-configs.js

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This script generates per-package tsconfig.*.json from the definition in
+ * package/package.json.
+ *
+ * Specifically,
+ * 1. If the package.json has fields `main`, `module` and `esnext`, targets
+ *    like ESM and ESNEXT tsconfig.json are generated. Otherwise only one
+ *    default CJS target is generated.
+ * 2. References in tsconfig.json are generated from the package.json fields
+ *    `dependencies`, `devDependencies` and `peerDependencies`.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const {
+  getDefaultTsConfig,
+  getEsmTsConfig,
+  getEsnextTsConfig,
+} = require('./update-ts-config-constants');
```

Comment:
Looks like a typo :slightly_smiling_face: 
```suggestion
} = require('./update-ts-configs-constants');
```